DHL/JSON/9df77954-633e-4a19-8e78-a2e73e34050f.json

```diff
@@ -0,0 +1,998 @@
+                                  "Nm": "Consent Category Description",
+                                  "Dsc": "Consent Category Desc",
+                                  "AttTy": "Constant",
+                                  "DflVal": "Captures Customs Consents from User"
+                                }
+                              ]
+                            },
+                            {
+                              "Id": "8430861b-38c6-4e53-92a0-5a100bdd0d05",
+                              "Nm": "Returned Shipment - Radio button",
+                              "Dsc": "Returned Shipment description",
+                              "isImg": false,
+                              "isExcl": false,
+                              "isMan": true,
+                              "Att": [
+                                {
+                                  "Id": "75e0a595-ab28-4e91-9949-fab52cdf82b5",
+                                  "Nm": "Returned Shipment Entity",
+                                  "Dsc": "Returned Shipment entity description",
+                                  "AttTy": "UserEntry",
+                                  "isExcl": false,
+                                  "isMan": true,
+                                  "UICmpt": {
+                                    "Order": 1,
+                                    "CmptNm": "RetrnShip",
+                                    "DisplayTtl": "",
+                                    "Tooltip": "",
+                                    "CmptTy": "RadioButton",
+                                    "DisplayDsc": {
+                                      "DisplayTxt": "Returned Shipment description"
+                                    },
+                                    "ValTy": "Standard"
+                                  },
+                                  "Enum": [
+                                    {
+                                      "Val": "Yes",
+                                      "DisplayTxt": "Yes",
+                                      "SrtOrd": 1,
+                                      "Nm": "Opt1"
+                                    },
+                                    {
+                                      "Val": "No",
+                                      "DisplayTxt": "No",
+                                      "SrtOrd": 2,
+                                      "Nm": "Opt2"
+                                    }
+                                  ]
+                                },
+                                {
+                                  "Id": "92011bf2-1fa7-40ab-95d7-d90a8081c2c7",
+                                  "Nm": "Purpose Name - Radio button",
+                                  "Dsc": "Purpose Name with a radiobutton component",
+                                  "AttTy": "Constant",
+                                  "DflVal": "Agree to parcel returned shipment after repair",
```

Comment:
change it "User agree to return parcel shipment after repair."

Reply:
Done